docs/spec_draft.md

```diff
@@ -388,6 +389,50 @@ dimensions `k` in `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.case
+
+### Semantics
+
+Returns the result of exactly 1 function from `branches` depending on the value
+of `index`. Formally, if $0 \le$ `index` $\lt$ `N-1`, result of executing
+`branches[index]` is returned, else, result of executing `branches[N-1]` is
+returned.
+
+### Inputs
+
+| Name       | Type                                         |
+|------------|----------------------------------------------|
+| `index`    | 1-dimensional tensor constant of type `si32` |
+| `branches` | a variable sized array of type `function`    |
+
+### Outputs
+
+| Name     | Type               |
+|----------|--------------------|
+| `result` | any supported type |
```

Comment:
variadic number of tensors of any supported types

Reply:
Tensors include 0-dimensional tensors (scalar). To be in sync with recent changes, let's make the `types` part singular.
`variadic number of tensors of any supported type`